### ISSUE TITLE
Add a Loading Container

### ISSUE THREAD

**Issue by vusters** [EnsembleUI/ensemble#716]
https://discord.com/channels/1031982848485359626/1091875561447641108/1138898040833458268

**Comment by vinothvino42** [comment]
@vusters @evshi What about this syntax for loading async task and showing results based on the state

```dart
AsyncView:
  api: getUser
  onIdle:
    Button:
       label: Get Account Summary
  onLoading:
    Shimmer:
      color: #ABCDEF
  onError:
    Text:
      text: ${getUser.error}
  onSuccess:
    AccountsSummary:
```

**Comment by vinothvino42** [comment]
NEW EDL:

```yaml
LoadingContainer:
  isLoading: ${myAPI.body == null}
  useShimmer: true
  widget: MyWidget
  loadingWidget: MyLoadingWidget
```

**Comment by vinothvino42** [comment]
@vusters @evshi PR - https://github.com/EnsembleUI/ensemble/pull/761

**Comment by vinothvino42** [comment]
It is now closed with #761 and #782 PR
